deps/openssl/openssl/util/libeay.num

```diff
@@ -75,7 +75,7 @@ BIO_new                                 78	EXIST::FUNCTION:
 BIO_new_accept                          79	EXIST::FUNCTION:
 BIO_new_connect                         80	EXIST::FUNCTION:
 BIO_new_fd                              81	EXIST::FUNCTION:
-BIO_new_file                            82	EXIST::FUNCTION:FP_API
+BIO_new_file                            82	EXIST::FUNCTION:
```

Comment:
That work worked like a charm. Thanks @bnoordhuis 
I'll update the PR with the changes later today.